src/targets/services/categorize.js

```diff
@@ -68,6 +74,16 @@ export const categorize = async () => {
     }
   })
   await client.saveAll(categorized)
+
+  await sendObservation({
+    client,
+    supervisorWebhook,
+    observationPayload: {
+      action: 'categorize',
+      operationsCategoriesBefore: operations.map(o => o.manualCategoryId),
+      operationsCategoriesAfter: operations.map(o => o.automaticCategoryId)
```

Comment:
I don't get the labels: how the manual categories corresponds to the "before" categories? Whatever the classification, the manual categories remain, right?